cmd/restic/cmd_backup.go

```diff
@@ -409,6 +410,11 @@ func collectTargets(opts BackupOptions, args []string) (targets []string, err er
 		return nil, nil
 	}
 
+	commentRe, err := regexp.Compile(`^(.*)+ #.*$`)
```

Comment:
I'm not confident in this regex. It should match _most_ use cases, but if people happen to have a space and a hash mark in the filename, e.g. `/media/pics/Super Cool Picture #42.jpg`, this would cause the file to be skipped, as the line would be set to `/media/pics/Super Cool Picture`.

I'd appreciate and am open to suggestions to improve this!